edgelet/contrib/config/linux/config.yaml

```diff
@@ -49,11 +71,21 @@
 #                information such as its IoT hub connection information.
 ###############################################################################
 
-# Manual provisioning configuration
+# Manual provisioning configuration using a connection string
 provisioning:
   source: "manual"
   device_connection_string: "<ADD DEVICE CONNECTION STRING HERE>"
```

Comment:
Do we want to recommend people to move to the new format for regular mode too? If so, change this too.

```
authentication:
  method: 'device_connection_string'
  device_connection_string: '...'
```

Reply:
I think there are several places that need to change including MS docs., current customer deployments, samples, tutorials and other external webpages that talk about Edge. So this might be a bit much to change this as the favored configuration at this stage to warrant for what is essentially a dev setup.